src/main/java/edu/cornell/kfs/fp/batch/service/impl/CreateAccountingDocumentServiceImpl.java

```diff
@@ -164,6 +164,14 @@ protected void createAndRouteAccountingDocumentFromXml(AccountingXmlDocumentEntr
             reportDetail.setSuccessfullyRouted(true);
             reportDetail.setDocumentNumber(document.getDocumentNumber());
             reportItem.getDocumentsSuccessfullyRouted().add(reportDetail);
+            if (!GlobalVariables.getMessageMap().getWarningMessages().isEmpty()
+                    && GlobalVariables.getMessageMap().getWarningMessages()
+                            .containsKey(CuFPConstants.CreateAccountingDocumentValidatedDataElements.PAYEE_NAME)) {
+                AutoPopulatingList<ErrorMessage> payeeError = GlobalVariables.getMessageMap().getWarningMessages()
+                        .get(CuFPConstants.CreateAccountingDocumentValidatedDataElements.PAYEE_NAME);
+                payeeError.stream()
+                        .forEach(em -> reportDetail.appendWarningrMessageToExistingWarningMessage(em.getMessageParameters()[0]));

```

Comment:
Should this section be reworked to construct the fully-formatted warning messages by calling MessageFormat.format() appropriately? Or do you specifically need to pick out the message parameters?

Reply:
I am not sure what you're asking here.  The message is being constructed on the report as I would expect.  This did get re-worked a bit based on previous code review comments

Reply:
This line is currently retrieving the first parameter from the error message and passing it to the append-warning-message method. (Not sure what rework you've done on it since then.) Should the line be changed to build the full message and pass that to the append-warning method?

Reply:
The first parameter of the error message is the the whole message, so it shouldn't need additional formating.